app/controllers/bars_controller.rb

```diff
@@ -3,7 +3,11 @@ class BarsController < ApplicationController
   before_action :require_access, only: [:edit, :destroy]
 
   def index
-    @bars = Bar.order(:name).page(params[:page]).per(5)
+    if params[:search]
```

Comment:
this works for now, but you could refactor this as i showed you in the clinic with `scope`. next time